_progress-bar.scss

```diff
@@ -0,0 +1,252 @@
+// progress-bar
+// ===
+
+// Configurable Variables
+// ---
+
+$progress-bar__progress-color: black !default;
+$progress-bar__success-color: $success-color !default;
+$progress-bar__error-color: $error-color !default;
+$progress-bar__background-color: $neutral-50 !default;
+
+
+// Dependencies
+// ---
+
+// Override Font Awesome's default path
+$fa-font-path: '/bower_components/font-awesome/fonts';
+
+@import 'stencil-variables/variables';
+@import 'level/level';
+@import 'font-awesome/scss/font-awesome';
+@import 'stencil-utils/dist/visibility/visibility';
+
+
+// Progress Bar root
+// ---
+
+.c-progress-bar {
+    display: flex;
+    align-items: center;
+    height: $tap-size;
+    margin: 0 15px;
+}
+
+
+// Radial Process Bar
+// ---
+
+.c-progress-bar__radial {
+    &:not(:empty) {
+        position: relative;
+
+        width: 45px;
+        height: 45px;
+        margin-right: 10px;
+    }
+
+    .c-progress-bar__radial-background {
```

Comment:
We strongly prefer stand-alone class selectors, even for sub-components. The exception is for sub-components of a modified component, e.g. `.c-card.c--fancy .c-card__title {…}`. Normally `.c-card__title` is a stand-alone selector – this follows a general principle of keeping selector specificity low to avoid getting snarled up with the cascade.